scheduler/planner.go

```diff
@@ -258,7 +258,7 @@ func (u *unitInfo) value() int64 {
 	// Increase the value for the number of dependents, so that
 	// tasks (and units) which block other tasks run before tasks
 	// that don't block other tasks.
-	value += priority * (u.NumDependents / length)
+	value += priority * (u.Settings.GetNumDependentsFactor() / 10) * (u.NumDependents / length)
```

Comment:
I feel like this shouldn't also consider priority, since we consider it above. Perhaps just multiplying num dependents by factor or something might be more straightforward (unless you have an idea why we consider overall length here)

Reply:
This is actually consistent with how priority behaves as a multiplicative factor. I don't think we would want this to be a special case for priority since the other factors account for priority.

Reply:
While I agree this function is hard to reason about, I think we should keep this line as is because it contains this PR to only changing the scalar effect of how many dependents a task has, rather than changing the logical formula of how the rank value is calculated.